pinot-controller/pom.xml

```diff
@@ -121,7 +122,7 @@
             </configuration>
           </execution>
           <execution>
-            <id>npm ci</id>
+            <id>npm ci</id><!-- npm ci removes node_modules! -->
```

Comment:
I have the feeling this comment is a bit strange now that we are actually running ci

Reply:
Removed.